weapon-config.inc

```diff
@@ -4520,6 +4520,18 @@ static ProcessDamage(&playerid, &issuerid, &Float:amount, &weaponid, &bodypart,
 			}
 		}
 	}
+	
+	// Check deagle damage
+	if (weaponid == WEAPON_CHAINSAW) {
```

Comment:
/facepalm
The weird thing was, it SOMEHOW worked.

Like, I made sure that the player's weapon skill was lower than 200 and they could damage me.

Also, I have accidentally reset my branch before where the progress went lost.